gobblin-core/src/main/java/gobblin/converter/json/JsonConverter.java

```diff
@@ -0,0 +1,77 @@
+package gobblin.converter.json;
+
+import gobblin.configuration.WorkUnitState;
+import gobblin.converter.Converter;
+import gobblin.converter.DataConversionException;
+import gobblin.converter.SchemaConversionException;
+import gobblin.converter.SingleRecordIterable;
+
+import java.io.IOException;
+import java.util.Map;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonNull;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+
+public class JsonConverter extends Converter<String, JsonArray, String, JsonObject> {
+
+  /**
+   * Take in an input schema of type string, the schema must be in JSON format
+   * @return a JsonArray representation of the schema
+   */
+  @Override
+  public JsonArray convertSchema(String inputSchema, WorkUnitState workUnit) throws SchemaConversionException {
```

Comment:
Its a simple conversion of the schema which is available in the pull file as string to as is JsonArray similar to what is done for CsvToJsonConverter.